i18n/en.pot

```diff
@@ -17,15 +17,18 @@ msgstr "Enter your password"
 msgid "Name"
 msgstr "Name"
 
-msgid "To use the Microplanning app, you need to log in to your Crosscut account."
-msgstr "To use the Microplanning app, you need to log in to your Crosscut account."
+msgid "To use the Microplanning app, you need to log in to your Crosscut account. "
+msgstr "To use the Microplanning app, you need to log in to your Crosscut account. "
+
+msgid "Don't have an account? "
+msgstr "Don't have an account? "
 
 msgid ""
-"Don't have an account? You can create one for free by clicking the 'Create "
-"Account' tab above or visit "
+" You can create one for free by clicking the 'Create Account' tab above or "
```

Comment:
Should there be a space at the start here, if `Don't have an account? ` has a space at the end?